docs/howto/add_relationship_links.md

```diff
@@ -0,0 +1,132 @@
+[Back to Guides](../README.md)
+
+# How to add relationship links
+
+AMS offers you many ways to add links in your JSON, depending on your needs.
+
+The following examples are in Rails but the logic is the same for any framework
+used.
+
+The following examples preassume that you don't want to load any data for relationships (include param is empty),
+specifically the following Rails controller was used for those examples:
+
+```ruby
+class Api::V1::UsersController
+  def show
+    render jsonapi: User.find(params[:id]),
+      serializer: Api::V1::UserSerializer,
+      include: []
+  end
+```
+
+### Links as an attribute of a resource
+**This is applicable to JSONAPI, JSON and Attributes adapters**
+
+You can just define an attribute in the resource, named `links` and return the
+links you want there. An example:
+
+```ruby
+class Api::V1::UserSerializer
+  attributes :id, :name, :links
+
+  def links
+    {
+      self: api_v1_user_path(object.id),
+      microposts: api_v1_microposts_path(user_id: object.id)
+    }
+  end
+end
+```
+
+This will resilt in (example is in jsonapi adapter):
+```json
+{
+    "data": {
+        "id": "1",
+        "type": "users",
+        "attributes": {
+            "name": "Example User",
+            "links": {
+                "self": "/api/v1/users/1",
+                "microposts": "/api/v1/microposts?user_id=1"
+            }
+        }
+    }
+}
+```
+
+
+### Links as an property of the resource definiton
+**This is only applicable to JSONAPI adapter**
+
+You can use the `links` helper method to define the links you need in the resource definition level. 
+
+```ruby
+class Api::V1::UserSerializer
+  attributes :id, :name
+
+  link(:self) {api_v1_user_path(object.id)}
+  link(:microposts) {api_v1_microposts_path(user_id: object.id)}
```

Comment:
our rubocop would want a space within the {}, e.g.: `{ api_v1_microposts_path(...) }`